packages/cli/e2e-test/helpers.js

```diff
@@ -95,18 +95,18 @@ async function waitForPageWithText(
   browser,
   path,
   text,
-  { intervalMs = 1000, maxAttempts = 240 } = {},
+  { intervalMs = 1000, maxLoadAttempts = 240 } = {},
 ) {
-  let attempts = 0;
+  let loadAttempts = 0;
   for (;;) {
```

Comment:
(not a real change request, just noting because loops are hard to make pretty sometimes)

now if this in the future were broken apart into waitForPage and waitForText, we could make beautiful regular for loops an not care about the loop variable and use return instead of break to exit etc etc :)

```js
async function waitForPage(browser, path, {...}) {
  for (const i = 0; i < maxLoadAttempts; ++i) {
    try {
      await browser.visit(path);
      return;
    } catch (error) {
      if (!error.message.match(EXPECTED_LOAD_ERRORS)) {
        throw error;
      }
      await new Promise(resolve => setTimeout(resolve, intervalMs));
    }

    throw new Error('Max attempts ...');
  }
}
```

Reply:
Right. This looks better. It should be a for loop instead of the current "while" loop.